env0/data_project_test.go

```diff
@@ -0,0 +1,60 @@
+package env0
+
+import (
+	"testing"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+var project = client.Project{
+	Id:          "id0",
+	Name:        "my-project-1",
+	CreatedBy:   "env0",
+	Role:        "role0",
+	Description: "A project's description",
+}
+
+func testProjectDataSource(t *testing.T, input map[string]string, mockFunc func(mockFunc *client.MockApiClientInterface)) {
+	resourceType := "env0_project"
+	resourceName := "test_project"
+	resourceFullName := dataSourceAccessor(resourceType, resourceName)
```

Comment:
```suggestion
	accessor := dataSourceAccessor(resourceType, resourceName)
```

to fit my new refactor pr